tests/utils.go

```diff
@@ -2058,7 +2063,16 @@ func NewRandomFedora32VMIWithFedoraUser() *v1.VirtualMachineInstance {
 }
 
 func NewRandomFedoraVMIWitGuestAgent() *v1.VirtualMachineInstance {
-	agentVMI := NewRandomVMIWithEphemeralDiskAndUserdata(ContainerDiskFor(ContainerDiskFedora), GetGuestAgentUserData())
+	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+
+	dnsServerIP, err := getClusterDnsServiceIP(virtClient)
+	PanicOnError(err)
+
+	searchDomains := getVMISeachDomains()
+	networkData := GetCloudInitNetworkData(ipv6MasqueradeAddress, ipv6MasqueradeGateway, dnsServerIP, searchDomains)
```

Comment:
I wonder if this is required for any test that creates a vmi with a guest agent?
If not, did you consider creating a separate function that would be used with specific tests or alternatively just add this data in a separate function as `AddUserData` does?

Reply:
I think it can be generalized, but for now it is good enough to configure IPv4/IPv6 VMI/s. Usually only an IPv6 one will need it now, because DHCPv4 is enabled by default anyway.

Reply:
It is indeed required for every vmi w/ guest agent created on an ipv6 cluster. 

It is innocuous for vmis on ipv4 clusters, and as such, we're just adding it as it.

Reply:
ok, fair enough I think.